code/datums/status_effects/debuffs.dm

```diff
@@ -670,7 +670,7 @@
 		return
 
 	playsound(debuff_owner.loc, "sound/bullets/acid_impact1.ogg", 4)
-	particle_holder.particles.spawning = 1 + (stacks * 2)
+	particle_holder.particles.spawning = max(stacks * 5, 15)
```

Comment:
This should probably be on add_stacks instead of tick.

Reply:
Yep